app/code/Magento/Ui/view/base/templates/stepswizard.phtml

```diff
@@ -62,7 +68,7 @@
         "*": {
             "Magento_Ui/js/core/app": {
                 "components": {
-                        "<?= $block->escapeJs($block->getComponentName()) ?>": {
+                        "<?= $escaper->escapeJs($block->getComponentName()) ?>": {
                             "component": "Magento_Ui/js/lib/step-wizard",
                             "initData": <?= /* @noEscape */ $this->helper(Magento\Framework\Json\Helper\Data::class)->jsonEncode($block->getInitData()) ?>,
```

Comment:
Does this even work? I think it misses a backslash at the beginning.
```
Magento\Framework\Json\Helper\Data::class
```

Reply:
It'd work both ways. If there is a leading backslash, then the path is absolute. Without a leading backslash, then the path is relative to the current namespace (or imported names via `use` statements). Given there is no namespace declared in this file, the default namespace of `\` is implied.